js/homework.js

```diff
@@ -0,0 +1,84 @@
+var bluChange = document.querySelector('#blueBtn');
+// select the button and assign to the bluChange variable
+bluChange.addEventListener('click', changeBlueColor);
+// create an Event and call the function
+function changeBlueColor() {
+    var myJumbo = document.querySelector('.jumbotron');
+    var myDonate = document.querySelector('.btn-primary')
```

Comment:
Try not to use classes as a selector if you are targetting an element that got an id. 

The reason is that a class can be repeated across the dom, while an id can only be once present.

Same applies with the other query selectors.